client/components/common/Meta/Html.vue

```diff
@@ -86,10 +88,10 @@ export default {
     border-color: currentColor;
   }
 
-  .quill-input-label {
+  .quill-label {
     position: absolute;
-    top: -23px;
-    color: rgba(0, 0, 0, 0.6);
+    top: -21px;
+    font-size: 14px;
```

Comment:
I'm ok with it if pixels are used elsewhere but in general, we shouldn't use them for font sizing.

Reply:
I've initially set changes in `rem`s but the figured resto of the style is in pixels so I changed back to pixels. 🤷‍♂️